examples/node-webhooks/README.md

```diff
@@ -0,0 +1,106 @@
+# Node Webhooks
+
+This app shows how to get webhook events on your Miro board using Node.js and Express.js.
+
+# 👨🏻‍💻 App Demo
+
+https://github.com/user-attachments/assets/1448b658-9e6f-4652-8300-6cebbf081f7a
+
+# 📒 Table of Contents
+
+- [Included Features](#features)
+- [Tools and Technologies](#tools)
+- [Prerequisites](#prerequisites)
+- [Associated Developer Tutorial](#tutorial)
+- [Run the app locally](#run)
+- [Folder Structure](#folder)
+- [Contributing](#contributing)
+- [License](#license)
+
+# ⚙️ Included Features <a name="features"></a>
+
+- [Miro Node Client Library with Express SDK](https://miroapp.github.io/api-clients/node/index.html)
+  - [miro.exchangeCodeForAccessToken()](https://miroapp.github.io/api-clients/node/classes/index.Miro.html#exchangeCodeForAccessToken)
+  - [miro.isAuthorized()](https://miroapp.github.io/api-clients/node/classes/index.Miro.html#isAuthorized)
+  - [miro.getAuthUrl()](https://miroapp.github.io/api-clients/node/classes/index.Miro.html#getAuthUrl)
+  - [miro.as()](https://miroapp.github.io/api-clients/node/classes/index.Miro.html#as)
+  - [api.getAllBoards()](https://miroapp.github.io/api-clients/node/classes/index.MiroApi.html#getAllBoards)
+
+# 🛠️ Tools and Technologies <a name="tools"></a>
+
+- [Node.js](https://nodejs.org/en)
+- [Express.js](https://expressjs.com/)
+
+# ✅ Prerequisites <a name="prerequisites"></a>
+
+- You have a [Miro account](https://miro.com/signup/).
+- You're [signed in to Miro](https://miro.com/login/).
+- Your Miro account has a [Developer team](https://developers.miro.com/docs/create-a-developer-team).
+- Your development environment includes [Node.js 14.13](https://nodejs.org/en/download) or a later version, and npm.
+- Your development environment includes [ngrok](https://ngrok.com/) or something similar.
+
+# 📖 Associated Developer Tutorial <a name="tutorial"></a>
+
+> To view a more in depth developer tutorial
+> of this app (including code explanations) see the [Getting started with webhooks tutorial](https://developers.miro.com/docs/getting-started-with-webhooks) on Miro's Developer documentation.
+
+# 🏃🏽‍♂️ Run the app locally <a name="run"></a>
+
+1. Create a new Miro app on [developers.miro.com](https://developers.miro.com/). This will take you to the app settings page. There you
+   will find the `MIRO_CLIENT_ID` and `MIRO_CLIENT_SECRET` to be added to your `.env` file. Ensure that `boards:read` scope is checked,
+   and then go ahead and install the app on your developer team. You will get an **access token** which you will need later to
+   authenticate the creation of your webhook subscription.
```

Comment:
```suggestion
1. **Create a Miro app** on [developers.miro.com](https://developers.miro.com/). This will take you to the app settings page, where you will find the `MIRO_CLIENT_ID` and `MIRO_CLIENT_SECRET`. These need to be added to your `.env` file.
   
   - Ensure the `boards:read` scope is checked.
   - Install the app on your developer team. You will get an **access token**, which is required later to authenticate your webhook subscription.
```